returnn/vocabulary.py

```diff
@@ -36,8 +45,14 @@ def tasks(self):
     def run(self):
         lex = lexicon.Lexicon()
         lex.load(self.bliss_lexicon.get_path())
-
-        vocab = {k: v for v, k in enumerate(lex.phonemes.keys())}
+        if isinstance(self.blacklist, tk.Path):
+            blacklist = uopen(self.blacklist.get_path()).readlines()
+            blacklist = [phoneme.strip() for phoneme in blacklist]
+        elif isinstance(self.blacklist, Set):
+            blacklist = self.blacklist
+        else:
+            blacklist = []
```

Comment:
also here maybe switch to empty set to have consistent type of blacklist?
```suggestion
            blacklist = set()
```